source/Missions/Messages/BoardGames/BoardGameMoveMessage.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Common.Messaging;
+using SandBox.BoardGames;
+using TaleWorlds.MountAndBlade;
+
+namespace Missions.Messages.Agents
+{
+    public readonly struct BoardGameMoveMessage : IEvent
+    {
+        public Move move { get; }
```

Comment:
nit: casing (Should be `Move` for a property)
This in turn lets you change `inMove` in ctor to `move`